server/services/caseloadService.ts

```diff
@@ -202,6 +205,8 @@ export default class CaseloadService {
               status: <LicenceStatus>licence.licenceStatus,
               type: <LicenceType>licence.licenceType,
               comUsername: licence.comUsername,
+              kind: <LicenceKind>licence.kind,
+              reviewDate: null,
```

Comment:
Has to be Integrated once review date field is added to Licence Summary in the backend